linters/bandit/plugin.yaml

```diff
@@ -13,6 +13,7 @@ lint:
       tools: [bandit]
       suggest_if: files_present
       direct_configs: [.bandit]
+      affects_cache: [pyproject.toml]
```

Comment:
You're right, my mistake. And we don't need to include .bandit here because the functionality of `direct_configs` is a superset of that of `affects_cache`. Will close this PR